### DISCUSSION TITLE
Add scaffold（脚手架）

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#78] [Ideas]
后面得加一个脚手架，帮助开发者快速基于 antd web3 创建一个 dapp，大家看看官方的脚手架应该用啥技术选型。

**Comment by afc163** [comment]
Next.js and Umi

**Reply by yutingzhao1991** [reply]
看上去是要搞两个，不搞 next.js 好像不够国际化，不搞 umi 好像也不对，国内用得还是多的，尤其是用 antd 的用 umi 的不少。

**Comment by yutingzhao1991** [comment]
@jeasonstudio @kiner-tang
